udata/templates/dataset/display.html

```diff
@@ -387,7 +389,9 @@ <h3 class="text-left">{{ _('Producer') }}</h3>
         <div class="row">
             <div class="col-sm-9 list-group resources-list smaller">
             {% for resource in dataset.community_resources %}
-                {% include theme('dataset/resource/list-item.html') %}
+                <div itemprop="hasPart">
```

Comment:
I tried with https://developers.google.com/structured-data/testing-tool/ and embedded concepts/properties looks to work as expected.